### PR TITLE
Made grammar/word-choice updates 🎓

### LINKED ISSUE #95: Grammar overhaul 📑
Before the official launch of `acmcsuf.com`, we need to scrutinize every sentence of the website.

This includes...

- Fixing grammar mistakes
- Making sense/good writing
- Checking for correct punctuation
- Correct capitalization

All instances in which grammar is ignored for the sake of the site's theme/design must be explained via an adjacent comment.

If your grammar exception is within HTML markup, follow the comment template below.

```html
<!-- GE: Explanation here... -->
```

If your grammar exception is within JavaScript, follow the comment template(s) below.

```js
// GE: Explanation here...
or
/* GE: Explanation here...
 * ...and here if you need extra...
 * ...space for your explanation...
 */
```

_Where `GE` stands for **G**rammar **E**xception._

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
Here is my best effort as far as professional language and word-choice goes. Please feel free to suggest alternative wordings or grammar corrections.

### CODE REVIEW

File: src/components/index/why-join.svelte [hunk 1]
```diff
@@ -7,14 +7,19 @@
...
-      strives to be a community that encourages growth, friendship, and technical
-      development to prepare people for challenges in their personal lives, school
-      lives, and careers. By joining <span class="brand-em">acm</span>, you'll
-      be stepping into a community that will help you get started on your tech
-      journey.
+      <span class="brand-em">acm<span class="brand-blue">CSUF</span></span>
+      is a community that encourages growth, friendship, and technical development.
+      We prepare each other to overcome future personal, academic, and professional
+      challenges that some may face down the road. By joining
+      <span class="brand-em">acm<span class="brand-em">CSUF</span></span>,
```
Comment (diamondburned) [inline src/components/index/why-join.svelte]: The "CSUF" part seems inconsistent with what's 4 lines above.

File: src/routes/about.svelte [hunk 2]
```diff
@@ -8,25 +8,30 @@
 <CommonHero>
   <h1 slot="title">about</h1>
   <h2 slot="headline">
-    what is acm<span class="brand-blue">CSUF</span>
+    What is acm<span class="brand-blue">CSUF</span>
```
Comment (diamondburned) [inline src/routes/about.svelte]: This part doesn't have a question mark, which seems inconsistent with what's above.

File: src/routes/connect.svelte [hunk 3]
```diff
@@ -11,29 +11,36 @@
 
 <CommonHero src="../assets/png/acm-csuf-badge.png" alt="acm-CSUF-Logo">
   <h1 slot="title">connect</h1>
-  <h2 slot="headline">follow us!</h2>
+  <h2 slot="headline">Follow us!</h2>
   <p slot="text">
     The best way to keep up with what's going on in the world of
     <span class="brand-bold">acm<span class="brand-blue">CSUF</span></span>
```
Comment (diamondburned) [inline src/routes/connect.svelte]: Shouldn't "acm" have `brand-em` here instead as well?

File: src/routes/connect.svelte [hunk 4]
```diff
@@ -11,29 +11,36 @@
...
+
 <CallToActionSection>
-  <h2 slot="headline">so what are you waiting for?</h2>
+  <h2 slot="headline">So what are you waiting for?</h2>
   <p slot="text">
-    Whatever the reason is for you to reach out to us, we can’t wait to hear,
-    and to getback to you! <span class="brand-bold"
-      >acm<span class="brand-blue">CSUF</span></span
+    We are excited to take your messages and ideas into consideration! <span
+      class="brand-bold">acm<span class="brand-blue">CSUF</span></span
```
Comment (diamondburned) [inline src/routes/connect.svelte]: Shouldn't "acm" have `brand-em` here instead?

File: src/routes/about.svelte (hunk 2 above)
Comment (diamondburned) [inline src/routes/about.svelte]: Shouldn't "acm" have `brand-em` here?

**Review by diamondburned — APPROVED** [review]

**Review by samuelsandoval1 — COMMENTED** [review]
Looks good. I believe acmCreate created the typography to be lowercase at the beginning of each sentence/header. So if you could keep to that that would be great. If mike says this way is fine then this is ok then!

**Review by samuelsandoval1 — CHANGES_REQUESTED** [review]
Make first letter lowercased

**Comment by EthanThatOneKid** [comment]
> Make first letter lowercased

I’ll lowercase the headings (using `text-transform: lowercase`) but I’ll keep the body text grammatically correct as it currently is.

**Review by samuelsandoval1 — APPROVED** [review]
LGTM!
Make sure to push to production

**Pull request merged**